### PR TITLE
Change GPL-3.0 license to Apache-2.0.

### PR DESCRIPTION
**Pull request by chaoticlonghair**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Cryptape Technologies LLC.
```
Comment (zhouyun-zoe) [inline AUTHORS]: Cryptape Technology LLC.

File: Cargo.toml [hunk 2]
```diff
@@ -1,11 +1,15 @@
 [package]
 name = "libsm"
 version = "0.3.0"
-authors = ["Tang Zongxun <tangzongxun@hotmail.com>", "Niu Junxiang <494521900@qq.com>"]
+authors = [
+    "Tang Zongxun <tangzongxun@hotmail.com>",
```
Comment (zhouyun-zoe) [inline Cargo.toml:5]: Zongxun Tang

File: Cargo.toml [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
+    "Niu Junxiang <494521900@qq.com>",
```
Comment (zhouyun-zoe) [inline Cargo.toml:6]: Junxiang Niu

File: Cargo.toml (hunk 2 above)
Comment (chaoticlonghair) [reply Cargo.toml:5]: I think, we should show some respect for the original authors.
Keep their name as they wrote.

Comment (zhouyun-zoe) [reply Cargo.toml]: Okey，I'm sold...

**Review by zhouyun-zoe — APPROVED** [review]

**Pull request merged**